### PR TITLE
set batman as primary and fix graph y axis.

### PR DESCRIPTION
**Pull request by pipermerriam**
Batman wins.  Lets make it the primary ranking algo.

Also, fix the y axis on the graphs.

enhanced-buzz-30878-1377041881-4

### CODE REVIEW

File: bughouse/static/js/player-rating-visualizations/views.js [hunk 1]
```diff
@@ -104,12 +104,9 @@ $(function(){
             };
         },
         getYBounds: function(lineData) {
-            if ( _.isUndefined(lineData) ) {
-                lineData = this.getRatingsData();
-            }
             return {
-                yMin: d3.min(lineData, function(d) { return d.y; }),
-                yMax: d3.max(lineData, function(d) { return d.y; })
+                yMin: 790,
```
Comment (nathanlubchenco) [inline bughouse/static/js/player-rating-visualizations/views.js]: :+1: 

probably a way to dynamically get this range, but this should do for quite awhile i would think

**Comment by nathanlubchenco** [comment]
lgtm

File: bughouse/static/js/player-rating-visualizations/views.js (hunk 1 above)
Comment (pipermerriam) [inline bughouse/static/js/player-rating-visualizations/views.js]: I started down that path and then went to this cause it was simple and got the job done.

Comment (nathanlubchenco) [inline bughouse/static/js/player-rating-visualizations/views.js]: yup, besides people with ratings below 790 probably don't want to see them
anyway ;)

On Tue, Mar 10, 2015 at 2:25 PM, Piper Merriam <notifications@github.com>
wrote:

> In bughouse/static/js/player-rating-visualizations/views.js
> <https://github.com/simpleenergy/bughouse-ranking/pull/28#discussion_r26162172>
> :
>
> >              return {
> > -                yMin: d3.min(lineData, function(d) { return d.y; }),
> > -                yMax: d3.max(lineData, function(d) { return d.y; })
> > +                yMin: 790,
>
> I started down that path and then went to this cause it was simple and got
> the job done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simpleenergy/bughouse-ranking/pull/28/files#r26162172>
> .
>



-- 
*Nathan Lubchenco* | Data Scientist | Simple Energy | M 720.470.8916

<http://simpleenergy.com/>
<http://www.bcorporation.net/community/simple-energy>

**Comment by pipermerriam** [comment]
This broke visualizations.   ignoring for now.

**Pull request merged**